garageScript/c0d3-app
# fix: Discord tokens disappear after connecting to discord

Final PR for #2345 

### Description

This PR change the query that the `AppNav` component uses to `getSession` query. To understand the reason of these changes, [please read the detailed issue description](#2345).

### Testing

1. Unlink discord if already connected — from `/profile/admin`
2. Log out from your c0d3 account  
3. Clear local storage
4. Login to your c0d3 account
5. Connect to discord
6. Refresh the page (/curriculum) multiple times 
7. Make sure you're still connected to discord
